source/simulator/stokes_matrix_free.cc

```diff
@@ -2396,14 +2396,13 @@ namespace aspect
     }
 
     // Setup the matrix-free operators
-    std::shared_ptr<MatrixFree<dim,double>> stokes_mf_storage(new MatrixFree<dim,double>());
-    matrix_free_objects.push_back(stokes_mf_storage);
+    std::shared_ptr<MatrixFree<dim,double>> matrx_free = std::make_shared<MatrixFree<dim,double>>();
```

Comment:
Can you fix `matrx_free` to `matrix_free`? Otherwise good to merge.

Reply:
That would help, right? Haha.